AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="cell_info_title">Neighboring Cells</string>
```

Comment:
@SecUpwN These strings here will be used in an en-US locale (aka american english)